ISSUE_TEMPLATE/stable.md

```diff
@@ -0,0 +1,97 @@
+First, verify that you meet all the [prerequisites](https://github.com/coreos/fedora-coreos-streams/blob/master/release-prereqs.md)
+
+Name this issue `stable: new release on YYYY-MM-DD` with today's date. Once the pipeline spits out the new version ID, you can append it to the title e.g. ` (31.20191117.3.0)`.
```

Comment:
Nit: spurious trailing space in version.

Also, :+1: on the sneaky example update.

Reply:
Heh, that starting space is actually on purpose. We want a space between the end of the current title and the opening parenthesis. Maybe I'm treating people too much like computers ;)

Reply:
Fixed!